app/StatusModule/WebHook.js

```diff
@@ -0,0 +1,49 @@
+var StatusModule = require('./StatusModule');
+
+/**
+ * WebHook
+ *
+ * @param {object} config
+ * @param {StatusManager} statusManager
+ * @constructor
+ */
+var WebHook = function(config, statusManager) {
+    StatusModule.call(this, config, statusManager);
+};
+
+/**
+ * Sets the http module.
+ */
+WebHook.prototype.init = function() {
+    this.https = require('https');
+};
+
+/**
+ * Handle the incoming statuses
+ */
+WebHook.prototype.handleStatus = function() {
+    var webHook,
+        request;
+
+    if (this.statusManager.hasStartedStatus()) {
+        webHook = this.config.started;
+    } else if(this.statusManager.hasFailureStatus()) {
+        webHook = this.config.failure;
+    } else {
+        webHook = this.config.success;
```

Comment:
Can you add checks to see if the config exists? If you only configure one endpoint the code might explode :innocent: